test/scalability/mzcompose.py

```diff
@@ -107,7 +110,9 @@ def run_with_concurrency(
             cursor.execute(connect_sql.encode("utf8"))
         cursor_pool.append(cursor)
 
-    print(f"Benchmarking workload {type(workload)} at concurrency {concurrency} ...")
+    print(
+        f"Benchmarking workload '{workload.__class__.__name__}' at concurrency {concurrency} ..."
```

Comment:
I prefer `type(workload)` over `workload.__class__`

Reply:
I can adjust it but just wanted to point out that this is not a refactoring but changes the output. `workload.__class__.__name__` returns the short name whereas `type(workload)` provides the fully qualified name. Though, no strong opinion, I can change it.

Reply:
Was not aware, fine then.